samples/apps/bookinfo/src/mongodb/ratings_data.json

```diff
@@ -1,2 +1,2 @@
 {rating: 5}
-{rating: 6}
```

Comment:
it's fine to have something different so you can visually tell which one you get